tests/layer_tests/pytorch_tests/test_atan2.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0i
+
+import pytest
+import torch
+import math
+import numpy as np
+
+from pytorch_layer_test_class import PytorchLayerTest
+
+class TestAtan2(PytorchLayerTest):
+    def _prepare_input(self, x, y, dtype=None):
```

Comment:
`_prepare_input` doesn't have this signature, if you want to use `kwargs` you need to provide them to `_test` call.